pkg/util/cache/basic_cache_test.go

```diff
@@ -37,6 +34,17 @@ func TestBasicCache(t *testing.T) {
 		assert.Equal(t, m[k], v)
 	}
 
+	wombat := "wombat"
+	initialTimestamp := c.GetModified()
+	c.modified = 0
+	c.Add("d", wombat)
+	cached, found := c.Get("d")
+	assert.True(t, found)
+	assert.Equal(t, cached, wombat)
+	// Timestamp was reset to 0 before calling .Add, so equality
+	// also means that the timestamp was updated
```

Comment:
I think this comment needs to be updated now that there's no equality check

Reply:
I removed the comment as it's more confusing than anything else.